charts/capi-runtime-extensions/templates/csi/aws-ebs/manifests/aws-ebs-csi-configmap.yaml

```diff
@@ -1045,7 +1045,12 @@ data:
     kind: ServiceAccount
     metadata:
       labels:
+        app.kubernetes.io/component: csi-driver
+        app.kubernetes.io/instance: aws-ebs-csi-driver
+        app.kubernetes.io/managed-by: Helm
```

Comment:
I noticed these Helm labels in other addons, I guess its not a problem for these to be here if not actually managed by helm? I also wonder if there is a helm controller if it tries to "adopt" them.

Something to test later.

Reply:
Yes that should make it possible. But yes good to test and see what happens.

Reply:
This is the kind of thing we should be able to do via e2e tests pretty easily I hope 🤞

Reply:
Opened #394